app/Platform/Services/GameListService.php

```diff
@@ -102,10 +102,12 @@ public function initializeGameList(array $gameIds): void
             $game['RetroRatio'] = $gameModel->points_total ? $gameModel->TotalTruePoints / $gameModel->points_total : 0.0;
 
             $game['ConsoleName'] = $this->consoles[$gameModel->ConsoleID]->Name;
-            $game['SortTitle'] = $game['Title'];
-            if (substr($game['Title'], 0, 1) == '~') {
-                $tilde = strrpos($game['Title'], '~');
-                $game['SortTitle'] = trim(substr($game['Title'], $tilde + 1) . ' ' . substr($game['Title'], 0, $tilde + 1));
+            $game['SortTitle'] = mb_strtolower($game['Title']);
+            if (substr($game['SortTitle'], 0, 1) == '~') {
+                $tilde = strrpos($game['SortTitle'], '~');
+                $game['SortTitle'] = trim(substr($game['SortTitle'], $tilde + 1) . ' ' . substr($game['SortTitle'], 0, $tilde + 1));
+            } else {
+                $game['SortTitle'] = " " . $game['SortTitle'];
             }
```

Comment:
I changed the `SortTitle` assignment logic.

"Tagged" games (eg: unlicensed, prototype, hack, etc) will now be moved to the bottom of the list in these datagrids. This mirrors the current _gameList.php_ behavior and seems desirable.

Reply:
Rather than prefixing a space on "good" titles to move them up, it might be more efficient to only prefix the modified titles with a "~" to move them down?

Reply:
Good idea, addressed in latest.